src/makeCalculator.js

```diff
@@ -37,7 +37,39 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const obj = {
+    result: 0,
+
+    add(result, value) {
+      return result + value;
+    },
```

Comment:
You already have a result in the upper scope
```suggestion
    add(a, b) {
      return a + b;
    },
```